src/test/java/com/kuzmins/service/TicketServiceTest.java

```diff
@@ -0,0 +1,273 @@
+package com.kuzmins.service;
+
+import static org.mockito.ArgumentMatchers.any;
+
+import com.kuzmins.model.tickets.Ticket;
+import com.kuzmins.model.tickets.TicketType;
+import com.kuzmins.model.users.User;
+import com.kuzmins.repositories.TicketRepository;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class TicketServiceTest {
+
+  @Mock private TicketRepository ticketRepository;
+
+  @InjectMocks private TicketService ticketService;
+
+  // void save(Ticket ticket)
+  @Test
+  void callSaveTicketRepositoryWhenCallSave() {
+    User user = new User();
+    Ticket ticket = new Ticket(user, TicketType.MONTH);
+
+    ticketService.save(ticket);
+
+    Mockito.verify(ticketRepository).save(ticket);
+    Mockito.verifyNoMoreInteractions(ticketRepository);
+  }
+
+  @Test
+  void doesNotSaveNullWhenCallSave() {
+    IllegalArgumentException illegalArgumentException =
+        Assertions.assertThrows(IllegalArgumentException.class, () -> ticketService.save(null));
+
+    Assertions.assertNotNull(illegalArgumentException);
+    Assertions.assertEquals("Ticket is null", illegalArgumentException.getMessage());
+
+    Mockito.verify(ticketRepository, Mockito.never()).save(null);
+    Mockito.verifyNoInteractions(ticketRepository);
+  }
+
+  @Test
+  void doesNotSaveInCaseExceptionWhenCallSave() {
+    User user = new User();
+    Ticket ticket = new Ticket(user, TicketType.MONTH);
+
+    Mockito.when(ticketRepository.save(ticket))
+        .thenThrow(new RuntimeException("Problem with DB access"));
+    RuntimeException runtimeException =
+        Assertions.assertThrows(RuntimeException.class, () -> ticketService.save(ticket));
+
+    Assertions.assertNotNull(runtimeException);
+
+    Mockito.verify(ticketRepository).save(any(Ticket.class));
```

Comment:
instead of `any(Ticket.class)` use `eq(ticket)`

Reply:
fixed